src/current/_includes/common/licensing-faqs.md

```diff
@@ -0,0 +1,171 @@
+1. CockroachDB emits [log messages]({% link {{ page.version.version }}/logging-overview.md %}) when a cluster is at risk of being [throttled](#throttling) due to license expiration or telemetry requirements. The database will also return notices to [SQL clients]({% link {{ page.version.version }}/cockroach-sql.md %}) or [applications]({% link {{ page.version.version }}/install-client-drivers.md %}) that try to execute [transactions]({% link {{ page.version.version }}/transactions.md %}).
+
+{{site.data.alerts.callout_info}}
+During the transition to the [CockroachDB Software License][csl], expiration behavior (including [throttling](#throttling)) will work as follows depending on your version of CockroachDB.
+
+For versions v23.1.29, v23.2.15, v24.1.7, v24.2.5, and v24.3.0 and later, clusters will behave as described in this documentation.
+
+For versions less than or equal to v23.1.28, v23.2.14, v24.1.6, and v24.2.4, the behavior will be as follows:
+
+- **Enterprise Trial** and **Enterprise Free** licenses will work as expected when you [set a license](#set-a-license).
+- However, the expiration behavior will be different. Clusters running these versions won't be [throttled](#throttling); instead, the Enterprise features will immediately stop working at license expiration.
+{{site.data.alerts.end}}
+
+## Renew an expired license
+
+To renew an expired **Enterprise** license, <a href="mailto:sales@cockroachlabs.com">contact Sales</a> and then [set](#set-a-license) the new license.
+
+To renew an expired **Enterprise Free** license, follow the steps in [Obtain a license](#obtain-a-license).
+
+## FAQs
+
+### Can I host CockroachDB as a service for internal use at my organization?
+
+Yes, employees and contractors can use your internal CockroachDB instance as a service, but no people outside of your organization will be able to use it without purchasing a license.
+
+### What constitutes hosting CockroachDB as a service?
+
+Hosting CockroachDB as a service means creating an offering that allows third parties (other than your employees and contractors) to operate a database. Specifically, third parties cannot modify table schemas.
+
+<a name="throttling"></a>
+
+### What is throttling and how does it work?
+
+When a cluster is being throttled, the number of concurrent open [SQL transactions]({% link {{ page.version.version }}/transactions.md %}) is limited to 5.
+
+This will happen in the following cases:
+
+- The cluster is not following telemetry requirements.
+    - There is a 7 day grace period for **Enterprise Free** and **Enterprise Trial** clusters to (re)start sending telemetry. This applies when the license is first [added](#set-a-license), or any time there is an interruption in telemetry.
```

Comment:
i'm hesitant to touch this and the `30-day` one because these have been legally reviewed

paranoid? maybe 

ps i hope i'm not coming off as disagreeable or anything